packages/parser/src/CombatData.ts

```diff
@@ -444,6 +446,41 @@ export class CombatData {
     }
 
     switch (event.logLine.event) {
+      case LogEvent.SPELL_DAMAGE_SUPPORT:
+      case LogEvent.SPELL_PERIODIC_DAMAGE_SUPPORT:
+      case LogEvent.RANGE_DAMAGE_SUPPORT:
+      case LogEvent.SWING_DAMAGE_SUPPORT:
+      case LogEvent.SWING_DAMAGE_LANDED_SUPPORT:
+        {
+          const supportAction = event as CombatSupportAction;
+          srcUnit.supportDamageIn.push(supportAction);
+
+          // [#sup1] This is a similar case to the SPELL_ABSORB note below
+          // If the first event of the match is a support event and the support spell caster
+          //  isn't in the units array yet we must add them first. HOWEVER
+          //  for support events we don't have the unitName!
+
+          // Due to this restriction we are currently skipping including any absorb events before the unit is ready
```

Comment:
interesting.. eventually we might want to have a mechanism for pushing events into a placeholder unit which later on gets more metadata as the actual unit shows up..